app/models/remote_theme.rb

```diff
@@ -84,8 +84,11 @@ def self.update_zipped_theme(
 
   # This is only used in the development and test environment and is currently not supported for other environments
   if Rails.env.test? || Rails.env.development?
-    def self.import_theme_from_directory(directory)
-      update_theme(ThemeStore::DirectoryImporter.new(directory))
+    def self.import_theme_from_directory(directory, update_components: false)
+      update_theme(
+        ThemeStore::DirectoryImporter.new(directory),
+        update_components: update_components,
```

Comment:
you can apply the "cool" new syntax there 😄 🏌️ 
```suggestion
        update_components:,
```